File: 1/js/TwoSum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target)
{
    for (var i = 0; i < nums.length; i++) {
        var idx = nums.indexOf(target - nums[i]);
        if (idx !== -1 && idx !== i) {
            return [i, idx];
        }
    }
};

File: 1/cs/TwoSum.cs
public class Solution
{
    public int[] TwoSum(int[] nums, int target)
    {
        for (int i = 0; i < nums.Length; i++)
        {
            int correspondingIdx = Array.IndexOf(nums, (target - nums[i]));
            if (correspondingIdx != -1 && correspondingIdx != i)
            {
                return new[] { i, correspondingIdx };
            }
        }
        return null;
    }
}